contracts/fixed_price_rev1.1.scilla

```diff
@@ -427,11 +464,29 @@ procedure HandleFulfillOrder(
   royalty_fee_bps <- & token_address.royalty_fee_bps;
   royalty_amount = portion sale_price royalty_fee_bps;
 
+  (* Calculating the Service Fee *)
   fee_bps <- service_fee_bps;
   svc_fee = portion sale_price fee_bps;
+  collection_id = zero_uint32;
+
+  (* Calculating the optional commission fee *)
+  collections_contract <- collection_address;
```

Comment:
@sam-zilliqa need to handle if `collection_address` is zero address (same as my comment above) .

Reply:
Currently its not possible to set the `collection_address` to the zero address. The reason is that this field here will disallow it. It will only accept an address that is a contract, and one that contains the specified fields
https://github.com/Zilliqa/marketplace-contracts/blob/3b62f7065b776cbd12129c1fcfd5a40e0bb063b6/contracts/fixed_price_rev1.1.scilla#L985